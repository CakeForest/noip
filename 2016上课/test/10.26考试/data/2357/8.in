3101676341792
