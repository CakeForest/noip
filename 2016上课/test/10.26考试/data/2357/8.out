3107475947520